backend-api/app/models/users.py

```diff
@@ -1,7 +1,13 @@
 from fastapi_users.db import SQLAlchemyBaseUserTableUUID
 
 from ..database import Base
+import sqlalchemy as sa
 
 
 class User(SQLAlchemyBaseUserTableUUID, Base):
-    pass
+    __tablename__ = "Users"
+
+    id = sa.Column(sa.Integer, primary_key=True, index=True)
+    email = sa.Column(sa.String, unique=True, index=True)
+    hashed_password = sa.Column(sa.String)
+    is_active = sa.Column(sa.Boolean, default=True)
```

Comment:
I think we should leave the fields in this model as is. It already contains an email, is active, and hashed password field. The id is a UUID.

However, I think changing the table name is fine.

Also, if we don't want UUID as the way to index users, we can use a different class than SQLAlchemyBaseUserTableUUID (https://fastapi-users.github.io/fastapi-users/12.1/configuration/databases/sqlalchemy/). I don't think we need to but we can discuss it if you want.

Reply:
<img width="710" alt="image" src="https://github.com/fydp-2024-55/fydp-2024.55/assets/49622234/a9c61790-f871-4445-8e29-8f45e35d6811">

Reply:
Understood, will make the suggested changes. Thanks!